WebKitLibraries/SDKs/appletvos16.0-additions.sdk/SDKSettings.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+        <key>CanonicalName</key>
+        <string>appletvos16.0-additions</string>
+        <key>IsBaseSDK</key>
+        <string>NO</string>
+        <key>FrameworkSearchPaths</key>
+        <array>
+            <string>System/Library/PrivateFrameworks</string>
```

Comment:
I think I can remove this search path. The projects which link against private frameworks already manually add it via SYSTEM_FRAMEWORK_SEARCH_PATHS.

Reply:
> I think I can remove this search path. The projects which link against private frameworks already manually add it via SYSTEM_FRAMEWORK_SEARCH_PATHS.

Turns out I cannot do this. I thought the build system would factor out a search path like

    SYSTEM_FRAMEWORK_SEARCH_PATHS = $(SDK_DIR)/System/Library/PrivateFrameworks

into multiple search paths — one for the base SDK and others for each sparse SDK. But it does not.